packages/logging/src/log.js

```diff
@@ -589,12 +589,27 @@ Log.prototype.write = function(entry, options, callback) {
     options = {};
   }
 
-  this.decorateEntries_(arrify(entry), function(err, decoratedEntries) {
-    // Ignore errors (the API will speak up if it has an issue).
+  if (!options.resource) {
```

Comment:
Oops. I intended to remove it all along, but it seems like in my quest to make the commits look nice I lost that change!

Removed now.